packages/opentelemetry-web/tsconfig.esm.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.esm.json",
+  "compilerOptions": {
```

Comment:
From what I see most of the packages have the same `compilerOptions` and `include` options could you move these parts to `tsconfig.base.esm.json` so that it can be removed from almost all packages leaving only 
```
{
  "extends": "../../tsconfig.base.esm.json"
}
```
WDYT ?

Reply:
Paths get resolved relative to the `tsconfig(.esm).json` they are defined in - so moving `include` to base causes:

> error TS18003: No inputs were found in config file '[location of repo]/packages/opentelemetry-api-metrics/tsconfig.esm.json'. Specified 'include' paths were '["../../src/**/*.ts"]' and 'exclude' paths were '["../../node_modules"]'.

Moving `outDir` or `tsBuildInfoFile` to base causes built files to be output to `[location of repo]/build/esm`

Likely also why every current `packages/*/tsconfig.json` does the same